embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/fakes/FakeCurrentSessionSpan.kt

```diff
@@ -22,10 +22,9 @@ internal class FakeCurrentSessionSpan : CurrentSessionSpan {
         return true
     }
 
-    override fun addAttribute(attribute: SpanAttributeData): Boolean {
-        addedAttributes.add(attribute)
-        return true
-    }
+    override fun addCustomAttribute(attribute: SpanAttributeData): Boolean = addedAttributes.add(attribute)
+
+    override fun removeCustomAttribute(key: String): Boolean = addedAttributes.removeIf { it.key == key }
```

Comment:
Isn't this API 24?

Reply:
I think I was mistaken. I've definitely seen lint errors for this before but it must be on another collection type.

Reply:
I did it without needing it. It's more readable anyway